### PR TITLE
add banner

### PR DESCRIPTION
**Pull request by Empino27**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,11 +5,38 @@
...
+        box-sizing: border-box;
+    }
+
+    body {
+        margin: 0
+    }
+
+    .banner {
+        padding: 50px 0;
+        background-color: whitesmoke;
```
Comment (Jayrald07) [inline index.html:27]: please change background

Comment (Empino27) [reply index.html:27]: Thanks. I'll change right away.

**Review by Jayrald07 — COMMENTED** [review]
Please change the color

**Review by Jayrald07 — APPROVED** [review]
This is okay.

**Pull request merged**